CMakePresets.json

```diff
@@ -0,0 +1,528 @@
+        "CMAKE_BUILD_TYPE": "Release",
+        "RE2C_BUILD_RE2GO": false
+      }
+    },
+    {
+      "name": "windows-msvc-release-ootree-full",
+      "displayName": "Windows x64: MSVC / Release / Out of tree (full)",
+      "inherits": "windows-msvc-release-ootree-fast",
+      "installDir": "${sourceDir}/install-2",
+      "cacheVariables": {
+        "RE2C_BUILD_RE2GO": true,
+        "RE2C_BUILD_LIBS": true,
+        "RE2C_REBUILD_LEXERS": true,
+        "RE2C_REBUILD_DOCS": false,
+        "RE2C_FOR_BUILD": "${sourceDir}/install/bin/re2c"
+      }
+    }
+  ],
+  "buildPresets": [
+    {
+      "name": "linux-build-base",
+      "hidden": true,
+      "configuration": "Debug",
+      "condition": {
+        "type": "equals",
+        "lhs": "${hostSystemName}",
+        "rhs": "Linux"
+      }
+    },
+    {
+      "name": "linux-gcc-ubsan-ootree-fast",
+      "displayName": "Linux x64: GCC / Ubsan / Out of tree (fast)",
+      "configurePreset": "linux-gcc-ubsan-ootree-fast",
+      "inherits": "linux-build-base",
+      "configuration": "Ubsan"
+    },
+    {
+      "name": "linux-gcc-ubsan-ootree-full",
+      "displayName": "Linux x64: GCC / Ubsan / Out of tree (full)",
+      "configurePreset": "linux-gcc-ubsan-ootree-full",
+      "inherits": "linux-build-base",
+      "configuration": "Ubsan"
+    },
+    {
+      "name": "linux-gcc-asan-ootree-fast",
+      "displayName": "Linux x64: GCC / Asan / Out of tree (fast)",
+      "configurePreset": "linux-gcc-asan-ootree-fast",
+      "inherits": "linux-build-base",
+      "configuration": "Asan"
+    },
+    {
+      "name": "linux-gcc-asan-ootree-full",
+      "displayName": "Linux x64: GCC / Asan / Out of tree (full)",
+      "configurePreset": "linux-gcc-asan-ootree-full",
+      "inherits": "linux-build-base",
+      "configuration": "Asan"
+    },
+    {
+      "name": "linux-gcc-debug-ootree-skeleton-fast",
+      "displayName": "Linux x64: GCC / Skeleton / Out of tree (fast)",
+      "configurePreset": "linux-gcc-debug-ootree-skeleton-fast",
+      "inherits": "linux-build-base"
+    },
+    {
+      "name": "linux-gcc-debug-ootree-skeleton-full",
+      "displayName": "Linux x64: GCC / Skeleton / Out of tree (full)",
+      "configurePreset": "linux-gcc-debug-ootree-skeleton-full",
+      "inherits": "linux-build-base"
+    },
+    {
+      "name": "linux-gcc-debug-intree-m32-skeleton-fast",
+      "displayName": "Linux x86: GCC / Skeleton / In tree (fast)",
+      "configurePreset": "linux-gcc-debug-intree-m32-skeleton-fast",
+      "inherits": "linux-build-base"
+    },
+    {
+      "name": "linux-gcc-debug-intree-m32-skeleton-full",
+      "displayName": "Linux x86: GCC / Skeleton / In tree (full)",
+      "configurePreset": "linux-gcc-debug-intree-m32-skeleton-full",
+      "inherits": "linux-build-base"
+    },
+    {
+      "name": "linux-gcc-release-ootree-skeleton-fast",
+      "displayName": "Linux x64: GCC / Release / Out of tree (fast)",
```

Comment:
Release presets are missing "Skeleton" in the `displayName` (configurations above as well).

Reply:
Please take a look: https://github.com/skvadrik/re2c/pull/374/commits/216932105275c3017cf0bf8bbb05c2492d09fa3d